src/gui/dialogs/title_screen.cpp

```diff
@@ -247,7 +247,7 @@ void title_screen::pre_show(window& win)
 	});
 
 	win.register_hotkey(hotkey::HOTKEY_FULLSCREEN, std::bind(fullscreen, std::ref(win.video())));
-	win.register_hotkey(hotkey::LUA_CONSOLE, std::bind(&launch_lua_console, std::ref(win)));
+	win.register_hotkey(hotkey::LUA_CONSOLE, std::bind(&launch_lua_console));
```

Comment:
No need to call `std::bind()` when you're not actually, you know, binding anything.